### ISSUE TITLE
cannot find module http-server when running monaca preview

### ISSUE THREAD

**Issue by wildabeast** [monaca/monaca-cli#65]
When I create a monaca app from the cli and run `monaca preview` I get `cannot find module http-server`. Running node 4.4.7, monaca 2.0.5.

Here's my full trace:

```
[ryan-sandbox]monaca create MonacaDemo
? Choose a category: Onsen UI
? Select a template - Press P to see a preview Onsen UI V2 JS Minimum

Downloading template...

Project created successfully.

Type "cd MonacaDemo" and run monaca command again.
  > monaca preview      => Run app in the browser
  > monaca debug        => Run app in the device using Monaca Debugger
  > monaca remote build => Start remote build for iOS/Android/Windows
  > monaca upload       => Upload this project to Monaca Cloud IDE
[ryan-sandbox]cd MonacaDemo/
[ryan-MonacaDemo]monaca preview
Installing packages. Please wait. This might take a couple of minutes.

http-server: module.js:327
http-server:     throw err;
http-server:     ^
http-server: Error: Cannot find module '/usr/local/lib/node_modules/monaca/src/serve/node_modules/http-server/bin/http-server'
http-server:     at Function.Module._resolveFilename (module.js:325:15)
http-server:     at Function.Module._load (module.js:276:25)
http-server:     at Function.Module.runMain (module.js:441:10)
http-server:     at startup (node.js:139:18)
http-server:     at node.js:968:3
```

and in case its useful, my monaca installation trace:

```
[ryan-MonacaDemo]npm install -g monaca
npm ERR! fetch failed https://registry.npmjs.org/monaca/-/monaca-2.0.5.tgz
npm WARN retry will retry, error on last attempt: Error: fetch failed with status code 400
\
> monaca@2.0.5 preinstall /usr/local/lib/node_modules/monaca
> node bin/preinstall


Checking for XCode ...
OK

Checking for XCode Command Line Tools ...
OK

Success: Confirmed that the dependent components are installed.
npm WARN deprecated npmconf@2.1.2: this package has been reintegrated into npm and is now out of date with respect to npm
npm WARN deprecated minimatch@1.0.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated graceful-fs@3.0.8: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.
/usr/local/bin/monaca -> /usr/local/lib/node_modules/monaca/bin/monaca
monaca@2.0.5 /usr/local/lib/node_modules/monaca
├── q@1.4.1
├── open@0.0.5
├── colors@0.6.2
├── xmldom@0.1.22
├── shelljs@0.3.0
├── optimist@0.6.1 (wordwrap@0.0.3, minimist@0.0.10)
├── monaca-inquirer@1.0.4 (ansi-escapes@1.4.0, mute-stream@0.0.6, through@2.3.8, cli-width@2.1.0, pinkie-promise@2.0.1, strip-ansi@3.0.1, figures@1.7.0, run-async@2.2.0, chalk@1.1.3, cli-cursor@1.0.2, string-width@1.0.1, rx@4.1.0, lodash@4.13.1)
├── monaca-lib@2.0.7 (padlock@1.1.2, qlimit@0.1.1, buffer-crc32@0.2.5, async@0.9.2, extend@2.0.1, lockfile@1.0.1, ncp@2.0.0, sse@0.0.6, tmp@0.0.26, watch@0.13.0, global-npm@0.3.0, glob@5.0.15, portfinder@0.4.0, rimraf@2.5.3, extract-zip@1.5.0, request@2.73.0, nconf@0.7.2, adbkit@2.4.1, xml2js@0.4.17)
└── cordova@5.4.1 (underscore@1.7.0, q@1.0.1, ansi@0.3.1, nopt@3.0.1, update-notifier@0.5.0, cordova-lib@5.4.1)
```

**Comment by frandiox** [comment]
@wildabeast Hello and thanks for the report! `http-server` should be installed the first time you run `monaca preview` since it's only used for that. Therefore it makes sense it does not appear in the original monaca installation trace.

I guess there were some permission errors when `http-server` was installed. We'll check it and try to reproduce it but meanwhile you can just install `http-server` manually with `npm install -g http-server`. I think that should fix your issues :)

**Comment by masahirotanaka** [comment]
@wildabeast Is the error still produce on your environment? We just released a new Monaca CLI (Version 2.1.0). Hope the new version will solve out.

**Comment by wildabeast** [comment]
Sorry for not following up guys! Got swamped in my github notification emails.

Anyhow I tried both installing http-server independently, as well as uninstalling and then installing 2.1.0 -- the error persisted. However since I seem to be alone in seeing this, I think we can safely say this is something wonky in my npm environment. Thanks both for the responses!
